tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "incremental": true,
     "target": "esnext",
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
Change all "module" and "target" to "es2017" and see if they work.

Reply:
It generates JavaScript code. However, these JavaScript files can cause errors in the xcm-demo because they are not in the CommonJS format.

I think we need to generate JavaScript code in the CommonJS format.

```
Users/star/devlab/xcm-demo/node_modules/@oak-network/types/index.js:1
import { jsonrpcFromDefs, typesFromDefs } from '@open-web3/orml-type-definitions/utils';
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

Reply:
so why `"module": "esnext"` worked and now needs to be changed to `commonjs`? What has changed?

`packages/types` is extending this tsconfig.json. It seems to me that this "module" value needs to be consistent with `packages/api-augment/tsconfig.json`.

Reply:
commonjs was the original value.